src/constants.js

```diff
@@ -399,6 +400,9 @@ export const TASK_COMPLETION_RULE_ANY: 'ANY_ASSIGNEE' = 'ANY_ASSIGNEE';
 export const TASK_EDIT_MODE_CREATE: 'CREATE' = 'CREATE';
 export const TASK_EDIT_MODE_EDIT: 'EDIT' = 'EDIT';
 
+/* ----------------- Task Validation ---------------- */
+export const TASK_MAX_GROUP_ASSIGNEES: 250 = 250;
```

Comment:
oooohhhh.. what is this? what does `:250` do here?

Reply:
using `: number` as type means any numeric value is valid as long as `typeof` is number. Making the type a literal value means two useful things: the constant is locked to that exact value, and flow (or typescript i guess?) will show the value in the IDE or complain more specifically if it mutates.

Reply:
👍